tools/npm-update

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh -eux
+
+# Development dependencies: See node_modules/README
```

Comment:
Could it point to the source in tools, so that it exists in a clean checkout also? Otherwise it's a bit of a circular dependency.

Also, do you mind renaming this to `npm-install`?

Finally, please remove the weird "fsdf" from the commit message. No crafting git SHAs! :grin:

Reply:
Sure.  Thanks for the suggestion.  (x2)